src/displayapp/widgets/Counter.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class Counter {
+      public:
+        Counter(int min, int max);
+
+        void Create();
+        static void upBtnEventHandler(lv_obj_t* obj, lv_event_t event);
```

Comment:
do we need the evenhandlers to be in the public interface? In Timer.cpp the button event handler is only in the cpp file